reference-architecture/osp-cli/ch4.8.4_install_docker_nodes.sh

```diff
@@ -0,0 +1,4 @@
+for H in $NODES
+do
+  ssh $H sudo yum -y install docker
```

Comment:
Actually, docker has a dependency on docker-storage-setup, so it will automatically run docker-storage-setup